timeless/views.py

```diff
@@ -329,6 +323,24 @@ def get(self, object_id):
                 return jsonify(response), HTTPStatus.OK
             abort(HTTPStatus.NOT_FOUND)
 
+        def post(self, payload):
+            """Fake response on post method."""
+            return jsonify(payload), HTTPStatus.OK
+
+        def put(self, payload):
+            """Fake response on put method."""
+            return jsonify(payload), HTTPStatus.OK
+
+        def delete(self, object_id):
+            """Fake response on delete method."""
+            if object_id == 5:
```

Comment:
@soin08 can you add this object_id as a class public constant named ``FAKE_OBJECT_ID``? Nobody from outside this class will really know that ``5`` is the reserved id for the fake object :D

Reply:
Done